docs/mkdocs.yml

```diff
@@ -23,7 +23,7 @@ theme:
   search_index_only: false
 
   highlightjs: true
-  hljs_languages: 
+  hljs_languages:
     - bash
     - cpp
```

Comment:
See comment on `cpp` vs `c++`. Does the mixed usage effect `hljs`?

Reply:
We probably should go with `cpp`, but I will just spent a few minutes thinking about it